common/src/main/java/io/homeassistant/companion/android/common/sensors/BatterySensorManager.kt

```diff
@@ -16,38 +16,32 @@ class BatterySensorManager : SensorManager {
             R.string.basic_sensor_name_battery_level,
             R.string.sensor_description_battery_level,
             "battery",
-            "%",
-            stateClass = SensorManager.STATE_CLASS_MEASUREMENT,
-            entityCategory = SensorManager.ENTITY_CATEGORY_DIAGNOSTIC
```

Comment:
we should keep these instead of removing :)

Reply:
Oops! Sorry